dockerfiles/storage/ubi.Dockerfile

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+
+FROM registry.redhat.io/ubi8/ubi-minimal:8.3
```

Comment:
same remarks than for the other Dockerfile (bash header, comment to see new updates and stick to a fine grain version tag)